src/api/dataops/dataOperationsModule.ts

```diff
@@ -42,14 +43,26 @@ export class DataOperationsModule implements IModule {
    * For TypeScript users it implements a generic type T that represents your dataset, default to any.
    * @template T Generic type of your dataset, default to any
    * @param dataset name of the dataset
+   * @param auth use specific authorization
    * @returns Dataset object used to fetch and modify data at your datasets.
    */
-  public dataset<T extends object = any>(dataset: string): Dataset<T> {
+  public dataset<T extends object = any>(dataset: string, auth?: string): Dataset<T> {
+    let config = this.injector.get(AuthOptions);
+    if (auth) {
+      config = {
```

Comment:
@marlosin Because it will create `auth: undefined` field in the config object in case if the second argument has been omitted which I wanted to avoid :)

Reply:
I meant instead of doing `config = { ...config, auth }` (keeping it inside the `if`). just trying to avoid creating a new object. or even `config.auth = auth` which accomplishes the same and easier to read.